### ISSUE TITLE
Detail page of remote endpoints throw 404

### ISSUE THREAD

**Issue by praveen-livspace** [TwiN/gatus#329]
### Describe the bug

Clicking on any of the remote endpoint on the main page takes to a 404 page.

### What do you see?

A blank detail page 

### What do you expect to see?

Detailed status of the remote endpoint.

### List the steps that must be taken to reproduce this issue

1. Start the application with a remote endpoint.
2. Click on the remote endpoint hyperlink.
3. Notice that the detail page does not show any data.

### Version

4.1.0

**Comment by TwiN** [comment]
Related: #64

**Comment by praveen-livspace** [comment]
@TwiN please check this PR #330 . It handles the case of remote endpoints

**Comment by m-emelchenkov** [comment]
Absolutely, it's a killer feature of Gatus (comparing e.g. to Uptime Kuma) and should works well. It's a shame to see that it hasn't been fully implemented. I see commit #330 with status "Closed with unmerged commits". Actually, I don't understand what this phrase means: work on the bug has been stopped?
